### DISCUSSION TITLE
Copilot no longer working when on new MacBook Pro

### DISCUSSION THREAD

**Discussion by scottybollinger** [community/community#17253] [Copilot Conversations] [answered]
Hi, been using Copilot for a while now. I got a new M1 mac and restored it from a time machine backup from the old working Macbook Pro and it no longer works. I can see a space for the icon on the toolbar but the icon is missing and there are no more suggestions in the code anymore. 

![image](https://user-images.githubusercontent.com/1869731/170177213-dadada51-15f6-463b-b615-9d1b7df98b33.png)

Only settings set for plugin are:

```JSON
  "github.copilot.enable": {
    "*": true,
    "yaml": false,
    "plaintext": true,
    "markdown": false
  }
```

I tried disabling the plugin and then deleting and reinstalling, with no luck. Any help greatly appreciated 🙏

**Comment by davecheney** [comment, marked as answer]
@scottybollinger your client is out of date, please upgrade. See https://github.com/github/feedback/discussions/16815

**Reply by davecheney** [reply]
Also, please upgrade your VSCode version and verify that you are using VSCode as VSChromium and OSS Code are not supported. Thank you.

**Comment by scottybollinger** [comment]
This fixed it. Thanks so much for the quick response! 🙏

**Comment by Salman-dott** [comment]
I am not insisting on giving the best answer, but I can, perhaps, provide the easiest one. I also have Mac and I am not great with it, to be honest. So I found [setapp.com](https://setapp.com/how-to/how-to-recover-an-unsaved-word-document-on-mac) that happened to give self-eplanatory answers to all of my questions. So I hope this will also help you. And I wish you all a great time and solving all computer-related problems.
